### ISSUE TITLE
Universal selector breaks raw directive

### ISSUE THREAD

**Issue by mgross1-godaddy** [elchininet/postcss-rtlcss#338]
minimal repro:
```css
/*rtl:raw:* { margin: 0;}*/
```
expected: 
```css
 [dir="rtl"] * { margin: 0;}
```
actual:
```css
/*rtl:raw:* { margin: 0;}*/
```

https://elchininet.github.io/postcss-rtlcss/#664e5db13ea64

**Comment by elchininet** [comment]
Hi @mgross1-godaddy,
I'll take a look at it.
Regards

**Comment by elchininet** [comment]
Hi @mgross1-godaddy,
I'll be fixed with [the next pull request](https://github.com/elchininet/postcss-rtlcss/pull/339).
Regards

**Comment by elchininet** [comment]
Hi @mgross1-godaddy,
The issue has been fixed in version [5.3.0](https://www.npmjs.com/package/postcss-rtlcss/v/5.3.0).
Regards

**Comment by mgross1-godaddy** [comment]
thank you!
